### ISSUE TITLE
MOBILE: Image of globe should be centered

### ISSUE THREAD

**Issue by iamjessklein** [mozilla/teach.webmaker.org#512]
center globe image

**Comment by iamjessklein** [comment]
This is on the Activities page.

**Comment by ScottDowne** [comment]
Some hints and background.

The globe is an image used in a component we created called Illustration.

A component is a custom html element, which we make with react.

The Illustration component is made to display an image to the left of some content for desktop, and displays the image above the content in mobile, which it centers the image on mobile by default.

So, an example:
```
          <Illustration
          src1x="/img/activities-page/svg/img-global-movement.svg"
          height={270} width={270}>
            <p>right side content</p>
          </Illustration>
```

src1x contains the src for non retina, and src2x is for retina. This is an svg though, so src2x is not needed.

height and width is mandatory, in pixels, of the size of the image. This speeds up the page load so the browser doesn't need to detect the image dimensions.

The `<p>` is whatever you want, and is the content to put with the image.

In the case of the globe, this is the Illustration element: https://github.com/mozilla/teach.webmaker.org/blob/develop/pages/activities.jsx#L253-L263

The globe should be centered by default, but it's not because we are overriding that in the activities page less, which is like css. Here is the override that's causing the issue: https://github.com/mozilla/teach.webmaker.org/blob/develop/less/components/activities-page.less#L60

If you remove that, the image for all the kits are also going to be centered. So what you actually want to do is make the selector for this ruling more specific, so it's only applied to `.activity-kit .illustration .image-container `

**Comment by iamjessklein** [comment]
fixed with #542 merge
